cmd/argo-watcher/main.go

```diff
@@ -211,7 +211,7 @@ func main() {
 
 	prometheusRegisterMetrics()
 
-	err := router.Run(":8080")
+	err := router.Run("localhost:8080")
```

Comment:
This should not be localhost. We can make it configurable, but let's leave the default as it was.

Reply:
I introduced HOST and PORT environment variables. They default to ":8080".